.github/workflows/docker_build_push.sh

```diff
@@ -25,7 +25,7 @@ if [[ "${GITHUB_EVENT_NAME}" == "pull_request" ]]; then
   PR_NUM=$(echo "${GITHUB_REF}" | sed 's:refs/pull/::' | sed 's:/merge::')
   LATEST_TAG="pr-${PR_NUM}"
 elif [[ "${GITHUB_EVENT_NAME}" == "release" ]]; then
-  REFSPEC="${GITHUB_REF}"
+  REFSPEC=$(echo "${GITHUB_REF}" | sed 's:refs/tags/::' | head -c 20)
```

Comment:
Can you change the head to 40? I added 20 kinda arbitrarily. Ive seen several branch names cut off.

Reply:
good point, for release could be hard to hit but it's close: 
```
echo 999.999.999rc100 | wc -c
17
```

I'll add 40 to the others also